README.md

```diff
@@ -122,6 +122,27 @@ await token.refresh((newToken) => { ... });
 </html>
 ```
 
+### Password-based authentication
+
+If you'd rather handle the authentication UI flow in your app, and when your oAuth2 client supports `grant_type: password`, you could request an access token in exchange for a user's credentials in one step:
+
+```javascript
+const client = new Client({
+  baseUrl: "https://staging-api.konto.io",
+  clientId: 'YOUR_CLIENT_ID',
+  redirectUri: "",
+  state: "",
```

Comment:
Let us make them optional in the `ClientOpts` type instead of putting empty strings into the constructor.

Reply:
changed